src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/API/ApprovalTestConfig.cs

```diff
@@ -1,7 +0,0 @@
-﻿using ApprovalTests.Reporters;
```

Comment:
This is a weird diff.   The file was moved to the folder `AssemblyConfiguration` without changes.  Unsure why this appears as it does.